src/tradelocker/tradelocker_api.py

```diff
@@ -1079,6 +1081,25 @@ def get_latest_asking_price(self, instrument_id: int) -> float:
         )
         current_ap: float = get_nested_key(current_quotes, ["ap"], float)
         return current_ap
+    
+    @log_func
+    @tl_typechecked
+    # TODO: should be replaced with "get_latest_asking_price"
```

Comment:
Please remove this comment, that's a leftover from the function "get_latest_asking_price".